docs/content/commands/npm-dist-tag.md

```diff
@@ -78,8 +78,8 @@ This command used to be known as `npm tag`, which only created new tags,
 and so had a different syntax.
 
 Tags must share a namespace with version numbers, because they are
-specified in the same slot: `npm install <pkg>@<version>` vs `npm install
-<pkg>@<tag>`.
+specified in the same slot: `npm install <pkg>@<version>` vs
+`npm install <pkg>@<tag>`.
```

Comment:
it might be easier if all these files weren’t hard-wrapped :-) they’re not meant for terminal viewing anyways.